test/cli/interactive.go

```diff
@@ -128,15 +162,18 @@ func (v *cmdInputOutput) InteractAndWait(cmd *exec.Cmd, handleErr errorHandler)
 		}
 	}
 
-	err := cmd.Wait()
-	v.console.TtyRaw().Close()
-
 	// Wait for end of stdout
-	if err := v.console.ExpectEOF(); err != nil {
-		handleErr(errors.Errorf("interaction: error when waiting for end of stdout: %w", err))
+	select {
+	case <-ctx.Done():
+		// skip waiting
+	default:
+		if err := v.console.ExpectEOF(expect.WithTimeout(EOFTimeout)); err != nil {
+			handleErr(errors.Errorf("interaction: error when waiting for end of stdout: %w", err))
+		}
```

Comment:
Na konci interakcie sa kontroluje, ze prikaz skoncil, a STDIN je teda tiez zavrety.
Tu som dal timeout `30s` ak by prikaz po dialogu robil nejaku dhlsiu pracu, tak nech sa to nemusi vzdy riesit zmenout timeoutu v `interaction.txt` + nechcel som vymyslat novy syntax na nastavovanie toho, ako dlho sa ma cakat na koniec (kedze to sa v `interaction.txt` nezadava).